blocks/overline-block/features/overline/default.test.jsx

```diff
@@ -1,10 +1,188 @@
 import React from "react";
-import { shallow } from "enzyme";
+import { useFusionContext } from "fusion:context";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import Overline from "./default";
 
-describe("Given Overline Block", () => {
-	it("should return an Overline element", () => {
-		const { default: Overline } = require("./default");
-		const wrapper = shallow(<Overline />);
-		expect(wrapper.find("Overline").length).toEqual(1);
+const mockContextObj = {
+	arcSite: "site",
+	globalContent: {
+		_id: "12345",
+		websites: {
+			site: {
+				website_section: {
+					_id: "/news",
+					name: "News",
+				},
+			},
+		},
+	},
+};
+
+jest.mock("fusion:properties", () => jest.fn(() => ({})));
+jest.mock("fusion:themes", () => jest.fn(() => ({})));
+jest.mock("fusion:context", () => ({
+	useFusionContext: jest.fn(() => mockContextObj),
+}));
+
+jest.mock("fusion:content", () => ({
+	useEditableContent: jest.fn(() => ({
+		editableContent: jest.fn(() => {}),
+	})),
+}));
+
+describe("overline feature for default output type", () => {
+	describe("when website_section content from globalContent is present", () => {
+		it("should render an a", () => {
+			const { container } = render(<Overline />);
+			expect(container.querySelectorAll("a")).toHaveLength(1);
+		});
+
+		it("should dangerously set the inner HTML to the website_section content", () => {
+			const { container } = render(<Overline />);
+			expect(container.querySelector("a").textContent).toContain("News");
```

Comment:
The test should not be querying the DOM to get the text. RTL allows you to query by text `screen.getByText('News')`

Reply:
Tests have been updated